test.py

```diff
@@ -175,7 +176,13 @@ async def add_test(self, shortname):
 
     async def run(self, test, options):
         try:
-            await test.run(options)
+            for i in range(1, 5):
```

Comment:
Please extract the `5` magic constant and give it a name.

Reply:
ok